docs/tools/index.md

```diff
@@ -245,7 +255,26 @@ Check out the following pre-built tools that you can use with ADK agents:
 
 </div>
 
-## Build your tools
+## Use pre-built tools with ADK agents
+
+Follow these general steps to include tools in your ADK agents:
+
+1. **Import:** Import the desired tool from the tools module. This is
+   `agents.tools` in Python, `google.golang.org/adk/tool/geminitool` in Go, or
```

Comment:
Should this be `google.golang.org/adk/tool` for Go since you're referring to the root tools module in each language, rather than a specific tool?